### ISSUE TITLE
[EBUTTD] metadata and copyright

### ISSUE THREAD

**Issue by kozmaz87** [ebu/ebu-tt-live-toolkit#278]
dropping all the metadata is probably the best thing to do. Some folk might get sniffy about dropping the ttm:copyright though! We possibly should keep that in if we can...

**Comment by EyalLavi** [comment]
`ttm:copyright` can appear in `<head>`only and has a cardinality of `0..1`. This means that only one instance is allowed in TT-D, but a Part 3 sequence may contain several. There was discussion in the F2F about relaxing the requirements for metadata so that multiple instances are allowed on body elements. How do we proceed?
- Suspend this issue until a metadata decision is made and the spec is updated
- Only preserve the most recent `ttm:copyright` from the part 3 document
- Drop `ttm:copyright` along with other metadata
- Merge all unique copyright elements, e.g. a comma-separated value

**Comment by nigelmegitt** [comment]
Go with "Merge all unique copyright elements, e.g. a comma-separated value" for now, as it's guaranteed to be valid.

**Comment by nigelmegitt** [comment]
This is none urgent, and we also are waiting on a spec decision. Bumping milestone.

**Comment by EyalLavi** [comment]
@nigelmegitt the decision was to relax metadata requirements. The approach we took is still valid. Can this issue be closed?

**Comment by nigelmegitt** [comment]
Not closed, but I've moved it to Nice to have.
